src/guides/v2.3/rest/tutorials/orders/order-add-items.md

```diff
@@ -113,6 +174,68 @@ The following example adds the downloadable product Advanced Pilates & Yoga (`sk
 }
 ```
 
+### Add a downloadable product to a cart (Links can be purchased separately)
```

Comment:
This information already exists [here](https://devdocs.magento.com/guides/v2.3/graphql/mutations/add-downloadable-products.html).

Reply:
The request parameter structure is different for mutations and rest do I really need to update the commit to point out mutation instead of rest.
CC: @keharper

Reply:
@Sairam9519, excuse me. I've missed up with checkout tutorials. My notice ties to GraphQL checkout tutorial 😊 

So, you PR is ok :)

Reply:
:)